src/psyclone/gocean1p0.py

```diff
@@ -2897,12 +2886,12 @@ def __init__(self, kernel_name, kernel_arg):
         access_mapping = api_config.get_access_mapping()
         try:
             access_type = access_mapping[access]
-        except KeyError:
+        except KeyError as err:
             valid_names = api_config.get_valid_accesses_api()
-            raise ParseError("Meta-data error in kernel {0}: argument "
-                             "access  is given as '{1}' but must be "
+            six.raise_from(ParseError("Meta-data error in kernel {0}: "
+                             "argument access  is given as '{1}' but must be "
```

Comment:
pycodestyle complains about the indenting.

Reply:
Done.